drivers/irqchip/sun4i-intc.c

```diff
@@ -93,12 +93,14 @@ sun4i_intc_register_irq(struct device *irqdev, struct device *dev,
 	vector->dev     = dev;
 	vector->handler = handler;
 
-	debug("IRQ %d now registered to device %s", irq, dev->name);
+	debug("%s: IRQ %d now registered to device %s",
```

Comment:
It's probably a good idea to consider doing this for other devices (i.e. `dm.c` and `sunxi-msgbox.c`).

Reply:
Ok, I will do those in separate commits.

Reply:
I did this for `sunxi-msgbox.c`, but I don't think this is necessary for `dm.c`, because it's not ambiguous what caused the messages.